cl/people_db/management/commands/cl_import_judges.py

```diff
@@ -300,6 +304,56 @@ def fix_fjc_positions(self, infile=None):
                         position.save()
                         add_items_to_solr.delay([p.pk], "people_db.Person")
 
+    def import_mag_bk_judges(self):
+        def read_mag_bk_csv(infile=None):
```

Comment:
Nested functions are usually a bad idea in Python unless you have a specific reason for doing them. These functions, unless I'm missing something aren't that special, so I'd just pull them out of the class altogether (if they don't rely on `self`, and thus you can), or I'd make them top-level methods on the class.

The idea is that when I read the main function, I don't want to see or think about those other functions. I just want to see the two things the main function does, and if I'm curious, I can go inspect the other two functions. By keeping them inside, they're still in the way of understanding the main purpose of the main function.

Reply:
Submitted another attempt.